src/cli/bin.ts

```diff
@@ -95,19 +166,58 @@ yargs
       inputPath,
       outputPath,
       serversPath,
+      fromVersion,
+      toVersion,
       removeWidgets: keysOfWidgetPluginsToRemove,
       pivotInputPath,
       debug,
       stack,
     }) => {
+      const migrationFunctions = [["4.3", migrateUIFolder], ["5.0"]];
```

Comment:
I'm not a fan of having the last element of the array being a different type: and the fact that it doesn't fit the `migrationFunctions` name reflects that.
Alternatives:
1. define a `latestMinorVersion` variable outside of the array
2. add `to` in the key, e.g. `"4.3_to_5.0"` instead of "4.3"

I like option 2. better, as it's understandable at a glance.

Reply:
I'm not a fan of 2. in the current state, because that means that we would have to parse the `"fromVersion_to_toVersion"` string to extract `fromVersions` and `toVersions`.
Since your comment was written, `migrateUIFolder` was renamed into `migrate_43_to_50`, and `migrationFunctions` typed as a tuple, so what the array represents is easily understandable IMO.

However I agree with not having the last element of the array being of a different type. I see two possibilities:
3. the last migration function is a no-op function 
4. add `to` to the key as you suggested in 2., but have another hardcoded `versions` array on the side to avoid the parsing of the string

I prefer 3., to avoid having two different arrays and have only one source of truth.
What do you think?

I haven't made any change for now

Reply:
Ah yes, I like the structure in 
```
{
  from: string;
  to: string;
  migrate: MigrationFunction;
}[]
```
Not sure about having each individual object exported from the corresponding migration function, I kind of like having a centralized array with all the versions and corresponding migration functions explicitly written.